MediaGallerySynchronization/Model/CreateAssetFromFile.php

```diff
@@ -83,6 +94,21 @@ public function execute(\SplFileInfo $file)
         );
     }
 
+    /**
+     * Return asset id if asset already exist by provided path
+     *
+     * @param string $path
+     * @return null|int
+     */
+    private function getAssetId(string $path): ?int
+    {
+        $asset = $this->getMediaGalleryAssetByPath->execute([$this->getPath($path)]);
+        if (!empty($asset)) {
+            return $asset[0]->getId();
+        }
+        return null;
+    }
+    
```

Comment:
```suggestion
    /**
     * Return asset id if asset already exist by provided path
     *
     * @param string $relativePath
     *
     * @return null|int
     * @throws ValidatorException
     */
    private function getAssetId(string $relativePath): ?int
    {
        $asset = $this->getMediaGalleryAssetByPath->execute([$relativePath]);
        return !empty($asset) ? $asset[0]->getId() : null;
    }
```

Reply:
Change `getPath` to `getRelativePath` because this name looks like what is going in the method :)